.gitignore

```diff
@@ -13,6 +13,7 @@ dist/
 
 # Build output (committed)
 !/dist
+!/package/dist
```

Comment:
All sorted, we now have the `packages` directory structure in this PR 🙌 

But because we can still run `npm publish` from `packages/govuk-frontend/dist` (with its own **package.json**) it keeps everything backwards compatible until we're ready for:

* https://github.com/alphagov/govuk-frontend/pull/3498

Reply:
This looks grand!

Should we do a release/pre-release dry run before merging to make sure the branches/files generate OK?

Reply:
Yeah we should

Since we're referring to `govuk-frontend` in lots of places we'll want to merge this one first:

* https://github.com/alphagov/govuk-frontend/pull/3558

Otherwise any support/prerelease branches (or merged release PRs) will have **package-lock.json** referencing the previous `"version"` string. First person to run `npm install` will see npm fix them all